vector/src/main/java/im/vector/dialogs/ExportKeysDialog.kt

```diff
@@ -49,7 +49,7 @@ class ExportKeysDialog {
 
             override fun afterTextChanged(s: Editable) {
                 when {
-                    TextUtils.isEmpty(passPhrase1EditText.text)                          -> {
+                    TextUtils.isEmpty(passPhrase1EditText.text) || passPhrase1EditText.length() < 8 -> {
```

Comment:
what's happen if you put some whitespaces in the passphrase ?

Reply:
they are counted